ArduPlane/quadplane.cpp

```diff
@@ -1926,6 +1941,11 @@ void QuadPlane::update_throttle_hover()
  */
 void QuadPlane::motors_output(bool run_rate_controller)
 {
+#if HAL_WITH_ESC_TELEM
```

Comment:
I'd much rather have this in QuadPlane::takeoff_controller(), and operate just like the existing check for tiltrotor.fully_up()
I don't think it should apply to manually piloted takeoffs
I don't think we should be calling set_spoolup_block() at all in quadplanes, we can just delay takeoff, and print a message as to why it is delayed